filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```diff
@@ -281,6 +281,10 @@ filebeat.inputs:
   # are matching any regular expression from the list. By default, no files are dropped.
   #prospector.scanner.exclude_files: ['.gz$']
 
+  # Include files. A list of regular expressions to match. Filebeat keeps only the files that
+  # are matching any regular expression from the list. By default, no files are dropped.
+  #prospector.scanner.include_files: ['.gz$']
```

Comment:
As we don't support gzip files yet, we should not use it as an example here. How about `/var/log/.*` ?